pkg/data/default.json

```diff
@@ -920,5 +920,40 @@
     "OnEnter": "publishWithDeferredConfirmOnEnter",
     "OnExit":"publishWithDeferredConfirmOnExit",
       "Path": "github.com/alibaba/opentelemetry-go-auto-instrumentation/pkg/rules/amqp091"
+  },
+  {
+    "Version": "[2.0.0,)",
+    "ImportPath": "github.com/apache/rocketmq-client-go/v2/producer",
+    "Function": "sendSync",
+    "ReceiverType": "*defaultProducer",
```

Comment:
[ReceiverType](https://github.com/alibaba/opentelemetry-go-auto-instrumentation/blob/main/docs/rule_def.md#instument-a-function) should be in form of regular expression, i.e. `\\*defaultProducer`